mybatis-spring-boot-samples/pom.xml

```diff
@@ -29,5 +29,6 @@
   <modules>
     <module>mybatis-spring-boot-sample-annotation</module>
     <module>mybatis-spring-boot-sample-xml</module>
+    <module>mybatis-spring-boot-sample-test</module>
```

Comment:
new module is not necessary just add new test for the existing ones. Sorry to not be clear.